### PR TITLE
Tests added for Array to and from Mapping

### PR DESCRIPTION
**Pull request by uda**
Added a few tests for main usage of from and to Mappings, to address issue #15.

(The full test failed due to removed method 'range' and the 'WebGuy' class)

### CODE REVIEW

File: tests/unit/ArrTest.php [hunk 1]
```diff
@@ -169,4 +169,271 @@ public function testExtract($arr)
...
+                        'field' => 'house',
+                        'val' => '221B',
+                    ),
+                ),
+            ),
+        );
+    }
+
+    /**
+     * @covers \Ohanzee\Helper\Arr::fromMapping()
```
Comment (shadowhand) [inline tests/unit/ArrTest.php]: This shouldn't be necessary. Coverage should be detected automatically.

File: tests/unit/ArrTest.php [hunk 2]
```diff
@@ -169,4 +169,271 @@ public function testExtract($arr)
             );
         $this->assertEquals(Arr::extract($arr['top'], $paths, $default), $expect);
     }
+
+    public function dataMappedContact()
+    {
+        return array(
```
Comment (shadowhand) [inline tests/unit/ArrTest.php]: We [require PHP 5.4](https://github.com/kohana/ohanzee-helpers/blob/master/composer.json#L30) so please use short array syntax.

File: tests/unit/ArrTest.php (hunk 1 above)
Comment (uda) [inline tests/unit/ArrTest.php]: OK, I just followed the other tests...

File: tests/unit/ArrTest.php (hunk 2 above)
Comment (uda) [inline tests/unit/ArrTest.php]: NP

**Comment by rjd22** [comment]
@shadowhand :+1:

**Pull request merged**